hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormat.java

```diff
@@ -56,6 +56,8 @@
 
   String UNKNOWN_WRITE_TOKEN = "1-0-1";
 
+  String DEFAULT_WRITE_TOKEN_METADATA_PARTITION = "0-0-0";
```

Comment:
Lets not please ever refer to some higher level functionality/stack i.e metadata in a lower level class. or member names/comments.